src/meta/src/hummock/hummock_manager.rs

```diff
@@ -261,10 +257,72 @@ where
                     .insert(compaction_group.group_id(), Levels { levels });
             }
             init_version.insert(self.env.meta_store()).await?;
```

Comment:
Since we will store this init_version after meta-service starts, why not load init_version from meta_store?

Reply:
> Since we will store this init_version after meta-service starts, why not load init_version from meta_store?

fixed

Reply:
> Since we will store this init_version after meta-service starts, why not load init_version from meta_store?

not fixed